lib/include/openturns/OTtestcode.hxx

```diff
@@ -991,7 +1000,7 @@ private:
 
     LOGTRACE(OSS() << "generating big sample...");
     const Sample sample(distribution_.getSample(momentsSamplingSize_));
-
+    sample.exportToCSVFile("sample_moments.csv");
```

Comment:
this was temporary for debug right ?

Reply:
Yes, I will leave it in the code the time to solve some of the bugs I detected

Reply:
fixed